k8s/chart/values.yml

```diff
@@ -15,3 +15,5 @@ ingress:
 
 secrets:
   secretEnv: ""
+
+project_name: pennlabs.org
```

Comment:
Shouldn't this also be replaced by startproject?

Reply:
@pawalt  said that the `values.yml` in the chart is global defaults for everything, but i'd defer to him on this one.

Reply:
Davis is correct on this one. Although, we should probably change this to example.org so that it doesn't conflict with the currently-active `pennlabs.org` deployment.

Reply:
This will just depend on the result on the thread above.